src/Razor/src/Microsoft.NET.Sdk.Razor/build/netstandard2.0/Microsoft.NET.Sdk.Razor.Compilation.targets

```diff
@@ -13,7 +13,7 @@ Copyright (c) .NET Foundation. All rights reserved.
 <Project ToolsVersion="14.0">
```

Comment:
@pranavkm can you look at this file? I found several missing changes so I added them. Also, updated this to the best of my knowledge to align with what is in Roslyn's master branch.